config/application.rb

```diff
@@ -41,7 +41,7 @@ class Application < Rails::Application
 
     # Locale configuration
     config.i18n.load_path += Dir[Rails.root.join("config", "locales", "**", "*.{rb,yml}")]
-    config.i18n.available_locales = %w[en mr-IN pa-Guru-IN bn-BD kn-IN en_BD en_IN en_ET en-IND bn-IN hi-IN ta-IN te-IN am-ET om-ET sid-ET so-ET ti-ET]
+    config.i18n.available_locales = %w[en mr-IN pa-Guru-IN bn-BD kn-IN en_BD en_IN en_ET en-IND bn-IN hi-IN ta-IN te-IN am-ET om-ET si_LK sid-ET so-ET ta_LK ti-ET]
```

Comment:
Ooooh fun decision time. Hyphens or underscores?
* Hyphens: Consistent with the other languages but constantly have to fight Transifex (the language mapping config in Transifex doesn't apply to the YML keys)
* Underscores: Inconsistent with current state but no Transifex friction

Is there a standard we should follow? Or are the locales we configure here arbitrary?

Reply:
I matched what Transifex submitted for the new languages here...is there any reason _not_ to do that, other than that inconsistency with the other locales?

Reply:
Not that I know of 🤔